runtime/doc/starting.txt

```diff
@@ -407,20 +407,18 @@ accordingly.  Vim proceeds in this order:
 	proceeding to load user configuration.
 
 4. Load user config (execute Ex commands from files, environment, …).
-	An environment variable (e.g. $VIMINIT) is read as one Ex command
-	line, where multiple commands must be separated with '|' or <NL>.
+	$VIMINIT environment variable is read as one Ex command line (separate
+	multiple commands with '|' or <NL>).
```

Comment:
this was changed c65d3fd67b0fb2688c444e43e54fb9a7d26f5f3d from the stronger, clearer form to a weaker form. Why? What "other" env vars are we talking about? VIMINIT is the only one that matters AFAIK.